parser/parser.go

```diff
@@ -144,12 +147,25 @@ func fileDigestMap(path string) (map[string]string, error) {
 		files = []string{path}
 	}
 
+	var mu sync.Mutex
+	var g errgroup.Group
+	g.SetLimit(max(runtime.GOMAXPROCS(0)-1, 1))
```

Comment:
Should this be `runtime.GOMAXPROCS(0)` ?

Reply:
Yes but I've noticed the system sometimes stutters with `runtime.GOMAXPROCS(0)` so it seems prudent to not use that value